cli/src/commands/next.rs

```diff
@@ -117,52 +79,24 @@ pub(crate) fn cmd_next(
             .view()
             .heads()
             .contains(current_wc_id);
-    let wc_revset = RevsetExpression::commit(current_wc_id.clone());
-    // If we're editing, start at the working-copy commit. Otherwise, start from
-    // its direct parent(s).
-    let start_revset = if edit {
-        wc_revset.clone()
-    } else {
-        wc_revset.parents()
-    };
-
-    let target_revset = if args.conflict {
-        start_revset
-            .children()
-            .descendants()
-            .filtered(RevsetFilterPredicate::HasConflict)
-            .roots()
-    } else {
-        start_revset.descendants_at(args.offset)
-    }
-    .minus(&wc_revset);
-
-    let targets: Vec<Commit> = target_revset
-        .evaluate_programmatic(workspace_command.repo().as_ref())?
-        .iter()
-        .commits(workspace_command.repo().store())
-        .try_collect()?;
+    let target = get_target_commit(
```

Comment:
The rest of `cmd_next()` and `cmd_prev()` look very similar. Also move that into the new module?

Reply:
Thanks for the nudge. Moved them all over now.